packages/commonwealth/client/scripts/views/pages/discussions/ThreadCard/ThreadCard.tsx

```diff
@@ -116,7 +117,7 @@ export const ThreadCard = ({
   const linkedSnapshots = filterLinks(thread.links, LinkSource.Snapshot);
   const linkedProposals = filterLinks(thread.links, LinkSource.Proposal);
 
-  const isStageDefault = isDefaultStage(thread.stage, customStages);
+  const isStageDefault = isDefaultStage(app, thread.stage, customStages);
```

Comment:
this might not work as expected in cases when thread card is used outside communities scope as `app?.chain?.meta?.custom_stages` would be undefined.

We can instead pass the `customStages` props to thread card, its already handled for non-community scope cases. For the remaining you'd need to pass `customStages={app?.chain?.meta?.custom_stages}` to thread card instances in community scope pages and then passing the whole app obj can be removed.

Reply:
This is the exact same logic as we had it previously so it will not break anything that was not already broken. The only difference here is that we pass it in the from the ThreadCard instead of importing it and using it in the helper file.

This is only needed for tests to pass due to some test login referencing this helper file and not having the DOM.